File: java/src/main/java/com/fengtian/qianer/atcoder/abc/abc156/ABC_156_A.java
package com.fengtian.qianer.atcoder.abc.abc156;

import java.util.Scanner;

public class ABC_156_A {
    public static void main (String[] args) {
        Scanner sc = new Scanner(System.in);

        final int N = sc.nextInt();
        final int R = sc.nextInt();

        System.out.println(N >= 10 ? R : R + 100 * (10 - N));
    }
}

File: kotlin/src/abc/abc156/ABC156_A.kt
package abc.abc156

import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)

    val n = sc.nextInt()
    val r = sc.nextDouble()
    val rating = if (n >= 10) r else r + 100 * (10 - n)
    println(rating.toInt())
}